scripts/ci/horti_entry.sh

```diff
@@ -30,8 +30,18 @@ echo Creating logs dir
 mkdir -p > /tests/logs/
 echo Installing Horti
 npm install -g horticulturalist 
-echo Check e2e servers is running
-curl http://localhost:31337/isRunning
-echo 
+echo restart is $RESTART
+echo tag is $IS_TAG
+if [ -n "$IS_TAG"  ] && [ -z "$RESTART" ]; then
+    echo 'Setting vars for tag'
+    UPGRADE=$BUILD
+    HORTI_BUILDS_SERVER=$DEFAULT_BUILDS_URL
+    BUILD='@medic:medic:release'
+fi
+
+echo 'Horti Env Vars'
+echo Build server url $HORTI_BUILDS_SERVER
```

Comment:
Looking at a line above: 
```
DEFAULT_BUILDS_URL=https://staging.dev.medicmobile.org/_couch/builds
...
HORTI_BUILDS_SERVER=$DEFAULT_BUILDS_URL
```

Just trying to make sure we won't accidentally log passwords and such, is this value always safe to echo?

Reply:
When using horti to install anything. We are using a separate READ_URL that should never have the password. The only one to have the password now is on publish.  The only thing changing is the endpoint.

Reply:
Excellent! Thanks for clarifying!